sdk/src/Services/S3/Custom/Transfer/_bcl+netstandard/TransferUtility.sync.cs

```diff
@@ -327,6 +327,34 @@ public void DownloadDirectory(TransferUtilityDownloadDirectoryRequest request)
                 ExceptionDispatchInfo.Capture(e.InnerException).Throw();
             }
         }
+
+        /// <inheritdoc/>
+        public TransferUtilityDownloadDirectoryResponse DownloadDirectoryWithResponse(string bucketName, string s3Directory, string localDirectory)
+        {
+            try
+            {
+                return DownloadDirectoryWithResponseAsync(bucketName, s3Directory, localDirectory).Result;
```

Comment:
Instead of using `.Result` use `.GetAwaiter().GetResult()` it handled exceptions better. As I recall with `.Result` you end up getting an `AggregateExcetion`.